devtools/conda-envs/test_env.yaml

```diff
@@ -32,6 +32,9 @@ dependencies:
   - nbsphinx
   - sphinx_rtd_theme
   - sphinxcontrib-bibtex ==1.0.0
+  
+  # Cosmetics
+  - rich
```

Comment:
I'm unsure about adding the `rich` package. Please take a look at this and change it if it needs to be placed somewhere else, thanks.